src/Components/Web/src/Head/HeadContent.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.AspNetCore.Components.Sections;
+
+namespace Microsoft.AspNetCore.Components.Head
```

Comment:
I was thinking it should be separate because @SteveSandersonMS suggested that the sections API stay internal for now. My concern was that a `Sections` namespace might be confusing if there are no "sections" classes being exposed. I'm happy to change this if you think that's better, though.